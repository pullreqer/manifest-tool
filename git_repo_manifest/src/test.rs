use crate::Manifest;
use insta::{assert_debug_snapshot, glob};
use quick_xml::de::{from_str, DeError};
use std::fs;
#[test]
fn run_test_inputs() {
    glob!("test_inputs/*.xml", |path| {
        let input = fs::read_to_string(path).unwrap();
        let foo: Result<Manifest, DeError> = from_str(&input);

        assert_debug_snapshot!(foo);
    });
}
