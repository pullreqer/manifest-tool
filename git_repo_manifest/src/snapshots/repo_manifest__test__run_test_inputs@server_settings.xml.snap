---
source: src/test.rs
expression: foo
input_file: src/test_inputs/server_settings.xml
---
Ok(
    Manifest {
        notice: None,
        manifest_server: None,
        remotes: [
            Remote {
                name: "origin",
                alias: None,
                pushurl: None,
                override: None,
                fetch: "https://github.com/",
                review: Some(
                    "https://github.com/",
                ),
                revision: None,
            },
        ],
        default: Some(
            DefaultTag {
                remote: Some(
                    "origin",
                ),
                revision: Some(
                    "master",
                ),
                dest_branch: None,
                upstream: None,
                sync_j: Some(
                    "1",
                ),
                sync_c: None,
                sync_s: None,
            },
        ),
        remove_projects: [],
        projects: [],
        extend_projects: [],
        repo_hooks: None,
        includes: [],
    },
)
